public/media_engagement/worldbank_plr_sms_traffic.html

```diff
@@ -0,0 +1,134 @@
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+    
+    <script src="https://www.gstatic.com/firebasejs/5.5.9/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/5.5.9/firebase-firestore.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.31.1/js/jquery.tablesorter.min.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/5.6.0/firebase-auth.js"></script>
+    <script type="text/javascript" src="../../public/coda/auth.js"></script>
+    <script type="text/javascript" src="firebase.js"></script>
+</head>
+<style>
+    .alert {
+        color: red;
+        font: 20px bold
+    }
+</style>
+<body>
+    <!-- NAVBAR WITH DROPDOWN & RESPONSIVE TOGGLE-->
+    <nav class="navbar navbar-expand-sm navbar-brown bg-white mb-3">
+        <div class="container">
+            <a class="navbar-brand" href="#"><img class="logo" src="avfdashboardlogo.PNG" alt="avf-logo"></a>
+
+            <!-- TOOGLE TRIGGER FOR SMALL DEVICES -->
+            <button class="navbar-light navbar-toggler" data-toggle="collapse" data-target="#navbarNav">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <!-- RESPONSIVE TOGGLE -->
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav ml-auto">
+                    <li><a class="nav-link border-left border-top border-right" href="https://avf-dashboards.firebaseapp.com/coda/coding_progress.html">MESSAGE CODING</a></li>
+
+                    <li class="nav-item dropdown">
+                        <!-- DROPDOWN TRIGGER -->
+                        <a href="#" class="nav-link border-top border-right active-link dropdown-toggle" data-toggle="dropdown">TRAFFIC</a> 
+                        <!-- DROPDOWN MENU -->
+                        <div class="dropdown-menu">
+                            <a class="dropdown-item active-item" href="https://avf-dashboards.firebaseapp.com/media_engagement/imaqal_media_engagement.html">IMAQAL</a> 
+                            <!-- <div class="dropdown-divider"></div> -->
+                        </div>
+                    </li> 
+
+                    <li class="px-2 pt-1 border-top border-right"><button class="btn btn-custom btn-sm shadow">Logout<i class="fas fa-sign-out-alt mx-2"></i></button></li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="container">
+        <!-- SMS traffic title & Timescale settings -->
+        <div class="d-md-flex justify-content-between p-1 border rounded">
+            <span class="txt-brown my-auto"><b>WORLDBANK_PLR</b></span>
+            <div>
+                <span class="align-content-end">Set the Timescale</span>
```

Comment:
I know you inherited this text, but while we're here I think just saying "Timescale" would be cleaner